(DOCSP-39500): Consolidate Install page (#3326)
mongodb/docs-realm: source/includes/api-details/javascript/install/install-supported-platforms-js-ts-description.rst

Supported Platforms
~~~~~~~~~~~~~~~~~~~

The installation instructions on this page assume you're using the JavaScript
library in a server-side environment. For instructions installing the library
for Internet of Things (IoT) devices, refer to :ref:`sdks-build-for-iot`.

Unsupported Platforms
~~~~~~~~~~~~~~~~~~~~~

- **Front-end Web Applications**: due to limitations of the browser environment,
  it is not possible to build a browser-based web app with this library. For
  front-end web applications, refer to :ref:`sdks-build-for-web`.

> - **Mobile App Development**: For cross-platform mobile app development, use
>   the :ref:`sdks-build-with-react-native`.

Reviewer: Wording is a bit odd when this gets expanded to the page title.

```suggestion
- **Mobile App Development**: For cross-platform mobile app development, refer
  to :ref:`sdks-build-with-react-native`.
```
